bootstrap/providers.php

```diff
@@ -4,10 +4,8 @@
     App\Providers\ActivityLogServiceProvider::class,
     App\Providers\AppServiceProvider::class,
     App\Providers\BackupsServiceProvider::class,
-    App\Providers\EventServiceProvider::class,
```

Comment:
I haven't double checked yet but removing the `EventServiceProvider` looks like a bad idea.
This will most likely break stuff like subuser email sending.

Reply:
Server subuser emailing seems to work just fine with it removed.
![image](https://github.com/pelican-dev/panel/assets/93738720/f7bb46e4-2259-4d1e-8d04-872e3e9d9b2e)

Reply:
Server subuser emailing seems to work just fine even with it removed.
![image](https://github.com/pelican-dev/panel/assets/93738720/8cb708ea-a055-44de-b301-1b4916477fa4)

Reply:
Is that a fresh install or did you upgrade from previous beta versions?

Reply:
Upgraded installation

Reply:
> Server subuser emailing seems to work just fine even with it removed. ![image](https://private-user-images.githubusercontent.com/93738720/346492919-8cb708ea-a055-44de-b301-1b4916477fa4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MjkzNDgsIm5iZiI6MTcyMDQyOTA0OCwicGF0aCI6Ii85MzczODcyMC8zNDY0OTI5MTktOGNiNzA4ZWEtYTA1NS00NGRlLWIzMDEtMWI0OTE2NDc3ZmE0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDA4NTcyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVjOTY0M2M1MjY3MjA2MDRmM2Q3Mzc0YWRkNmQwYTVmMDQzMGYyZGFhOTlmZWIxMjdiMWQzNWM5NzVhNGNlYzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7OieTKVooqxmGKTDIAa3_NhVSvhCr4XnOQ_jg75_Fe4)

Okay, figured it out. This is the "Account created" email but you didn't get the "Added to server" email, right?
That's the email that is missing.